app/upgrades.go

```diff
@@ -55,7 +55,17 @@ var handlers = map[string]appUpgrade{
 			return app.UpgradeKeeper.GetModuleVersionMap(ctx), nil
 		},
 	},
-	"quicksilver-rc1": { // upgrade for v1.15.0-rc1
+	"quicksilver-rc1": { // upgrade for v1.15.0-rc2
+		Handler: func(ctx sdk.Context, app *App, plan upgradetypes.Plan) (module.VersionMap, error) {
+			app.MarkerKeeper.RemoveIsSendEnabledEntries(ctx)
+			app.AttributeKeeper.PopulateAddressAttributeNameTable(ctx)
+			versionMap := app.UpgradeKeeper.GetModuleVersionMap(ctx)
+			ctx.Logger().Info("Starting migrations. This may take a significant amount of time to complete. Do not restart node.")
+			return app.mm.RunMigrations(ctx, app.configurator, versionMap)
+		},
+	},
+	"quicksilver-rc2": {}, // upgrade for v1.15.0-rc3
```

Comment:
Historically, the upgrade rc numbers have been bumped independently of the `provenanced` rc numbers. And since `quicksilver -rc1` was for `v1.15.0-rc2`, this one already doesn't match up.